autopush/src/server/mod.rs

```diff
@@ -639,9 +639,12 @@ impl Future for MegaphoneUpdater {
                             }
                         }
                         Ok(Async::NotReady) => return Ok(Async::NotReady),
-                        Err(_) => {
-                            // TODO: Flag sentry that we can't poll megaphone API
-                            debug!("Failed to get response, queue again");
+                        Err(error) => {
+                            error!("Failed to get response, queue again {:?}", error);
+                            capture_message(
```

Comment:
Sentry also has a `capture_error_chain` that might work (though I've never tried it..)